.github/workflows/main.yml

```diff
@@ -27,16 +27,16 @@ jobs:
         run: cd ./docusaurus && yarn install --force
 
       - name: Validate TypeScript
-        run: yarn typescript && yarn typescript:example
+        run: yarn typescript && yarn typescript:control-app
 
       - name: Validate ESLint
-        run: yarn lint && yarn lint:example
```

Comment:
I would like to mention that this is either a bug or obsolete as far as I can understand. Running eslint checks all folders so running it again from a child folder in structure just runs another eslint for that directory only. 

Please check the pattern we're using for eslint in the attached screenshot

<img width="664" alt="Screenshot 2023-03-09 at 17 07 45" src="https://user-images.githubusercontent.com/18323849/224083794-0214f286-932b-4ed9-8b38-969a89e2b54b.png">